recipes/libtins/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_INSTALL_LIBDIR=lib -DLIBTINS_BUILD_SHARED=0 ..
```

Comment:
We prefer shared linkage on conda-forge, thus `BUILD_SHARED` should be on?
```suggestion
cmake ${CMAKE_ARGS} -DCMAKE_BUILD_TYPE=Release -DLIBTINS_BUILD_SHARED=ON ..
```

Reply:
fixed